### PR TITLE
Move pytest tests out of the notebook and test with Travis

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

File: tests/test_data.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+from bods_validate import this_dir, format_checker, absolute_path_to_schema_dir, resolver
+from bods_validate import MissingStatementTypeError
+from bods_validate import bods_validate_package, bods_validate_statement
+
+
+@pytest.mark.parametrize('json_path', [
+    'data/entity-statement/valid/valid-entity-statement.json',
+    'data/person-statement/valid/valid-person-statement.json',
+    'data/beneficial-ownership-statement/valid/valid-beneficial-ownership-statement.json',
+    pytest.mark.xfail('../examples/flat-serialisation/gb-coh-entity-statement.json'),
```
Comment (ScatteredInk) [inline tests/test_data.py]: This example should be fixed in latest commit, so we'd expect it to pass now.

File: tests/test_data.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    '../examples/flat-serialisation/gb-coh-person-statement.json',
```
Comment (ScatteredInk) [inline tests/test_data.py]: This example should also be fixed.

File: tests/test_data.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+def test_valid_statement_json(json_path):
+    with open(os.path.join(this_dir, json_path)) as f:
+        json_data = json.load(f)
+    bods_validate_statement(json_data)
+
+
+@pytest.mark.parametrize('json_path', [
+    'data/bods-package/valid/valid-bods-package.json',
+    '../examples/flat-serialisation/gb-coh-bods-package.json',
+    pytest.mark.xfail('../examples/flat-serialisation/single-direct.json'),
```
Comment (ScatteredInk) [inline tests/test_data.py]: This example has been removed in latest commit, pending an updated version.

File: tests/test_data.py (hunk 1 above)
Comment (Bjwebb) [reply tests/test_data.py]: I've rebased onto 0.1-dev, but this still seems to fail.

Comment (ScatteredInk) [reply tests/test_data.py]: Ah, thanks, that should be fixed now.

**Comment by Bjwebb** [comment]
Thanks @ScatteredInk, the xfails are gone now.

**Pull request merged**